RadarSDK/Radar.m

```diff
@@ -849,4 +873,52 @@ - (void)dealloc {
     [[NSNotificationCenter defaultCenter] removeObserver:self];
 }
 
+
++ (BOOL)isTestKey {
```

Comment:
can this be internal/not part of the public API?

Reply:
yeah thinking somewhere like `RadarUtils` might be a better home for this fn, especially if we want to use it in other places - had a similar comment about `asBool` that @jrtibbetts added to the PR that exposed `user.mocked`

Reply:
i moved this to private (not included in Radar.h) to match android 
https://github.com/radarlabs/radar-sdk-android/blob/master/sdk/src/main/java/io/radar/sdk/Radar.kt#L2198